src/coverup/coverup.py

```diff
@@ -112,7 +112,7 @@ def positive_int(value):
 def test_file_path(test_seq: int) -> Path:
     """Returns the Path for a test's file, given its sequence number."""
     global args
-    return args.tests_dir / f"test_{PREFIX}_{test_seq}.py"
+    return args.tests_dir / f"test_{PREFIX}_{args.model.split('/')[0]}_{test_seq}.py"
```

Comment:
How about instead we add a `--prefix` that replaces the current `PREFIX` code? Then we could invoke it as `--model foo/bar!123 --prefix coverup_foobar` without having to worry about invalid characters in the model name.

Reply:
Works for me